README.md

```diff
@@ -1,29 +1,52 @@
-# Faasm Experiments - LAMMPS
+# LAMMPS experiment
 
-LAMMPS is a framweork for efficiently computing short-range molecular dynamics.
```

Comment:
The `README` for experiments should contain full instructions on how to build the code, set up any necessary data, and run the experiments.